Userland/Applications/PixelPaint/MainWidget.cpp

```diff
@@ -747,7 +747,8 @@ void MainWidget::initialize_menubar(GUI::Window& window)
     m_layer_menu->add_action(*m_move_active_layer_down_action);
 
     m_layer_menu->add_separator();
-    m_layer_menu->add_action(GUI::Action::create(
+
+    m_remove_active_layer_action = GUI::Action::create(
         "&Remove Active Layer", { Mod_Ctrl, Key_D }, g_icon_bag.delete_layer, [&](auto&) {
             auto* editor = current_image_editor();
             VERIFY(editor);
```

Comment:
The `if (!active_layer) return;` below should probably become `VERIFY(active_layer);` since this action should never run when there is no active layer.